src/Pet/PetController.js

```
@@ -0,0 +1,36 @@
+import express from 'express';
+import {validationResult} from 'express-validator';
+import service from './PetService';
+import {AuthRequired} from '../Utils/AuthUtils';
+
+const app = express();
+
+app.post('/createPet', AuthRequired, async (req, res) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(422).json({errors: errors.array()});
+  }
+  const {email} = req.user;
+  const response = await service.PetCreate(
+    email,
+    req.body.category,
+    req.body.special_needs,
+    req.body.diet,
+    req.body.name,
+  );
+  return res.json(response);
+});
+
+app.get('/getPet', AuthRequired, async (req, res) => {
+  const response = await service.SelectAllPets(req.user);
```

So if there are no pets found through the query do you want to throw an error or let client side check?